Test/Core/Utility/GarbageCollectionUtils.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using NUnit.Framework;
+
+namespace Duality.Tests.Utility
+{
+	public class GarbageCollectionUtils
```

Comment:
Since this is a new utility class, please add XML docs to class and at least its public members.

Reply:
added docs